packages/react-sdk/src/state/crdt/documents/elements.ts

```diff
@@ -77,6 +86,7 @@ const shapeElementSchema = elementBaseSchema
     textBold: Joi.boolean(),
     textItalic: Joi.boolean(),
     textSize: Joi.number().strict(),
+    textFontFamily: Joi.string().strict(),
```

Comment:
I wonder if we should just default to `Inter` here. Then we can save a lot of undefined cases, because we know something is there. But no strong opinion here…

Reply:
If `Inter` is now the default, we can remove all the undefined checks.